misk/src/main/kotlin/misk/client/HttpClientFactory.kt

```diff
@@ -13,6 +13,7 @@ import javax.net.ssl.X509TrustManager
 class HttpClientFactory {
   @Inject lateinit var sslLoader: SslLoader
   @Inject lateinit var sslContextFactory: SslContextFactory
+  @com.google.inject.Inject(optional = true) lateinit var envoyEndpointConfiguration: EnvoyEndpointConfiguration
```

Comment:
Is there any weirdness that can occur from us using two different Inject annotations?

Reply:
mixing doesn't cause any issues AFAIK